src/Chapter2.hs

```diff
@@ -620,7 +627,18 @@ Write a function that takes elements of a list only on even positions.
 >>> takeEven [2, 1, 3, 5, 4]
 [2,3,4]
 -}
-takeEven = error "takeEven: Not implemented!"
+takeEven :: [a] -> [a]
+takeEven [] = []
+takeEven [x] = [x]
+takeEven (x : _ :xs) = x : takeEven xs
+
+takeEvenWithGo :: [a] -> [a]
+takeEvenWithGo = go True
+  where
+    go :: Bool -> [a] -> [a]
+    go _ [] = []
+    go True (x:xs) = x : go False xs
+    go False (_:xs) = go True xs
```

Comment:
Very creative! 🎨 
Great initiative to play with different approaches, this is really helpful 👏🏼 
Nice choice of `Bool` as a state, as this represents it very accurately 👍🏼